tendrl/commons/jobs/__init__.py

```diff
@@ -314,7 +303,7 @@ def process_job(jid):
                        "job status invalid"
                 raise FlowExecutionFailedError(_msg)
             else:
-                job = job.load()
```

Comment:
Sometimes job.load() is not updating object with new values

Reply:
This will need a exhaustive testing of multiple jobs.

Reply:
i will do that